### PR TITLE
Add contentDescription to the images and icons

### PR DESCRIPTION
**Pull request by rkowase**
## Issue
- none

## Overview (Required)
- Add contentDescription

## Links
- none

## Screenshot
none

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <string name="nav_item_all_survey">Conference survey</string>
     <string name="nav_content_description_drawer_open">Open navigation drawer</string>
     <string name="nav_content_description_drawer_close">Close navigation drawer</string>
+    <string name="nav_content_description_drawer_header">DroidKaigi 2018</string>
```
Comment (takahirom) [inline app/src/main/res/values/strings.xml]: Can you add japanese? Probably It will warning 🇯🇵

Comment (rkowase) [reply app/src/main/res/values/strings.xml]: Oops, I'll add.

**Comment by takahirom** [comment]
You sent pull request 9 times? 
Are you machine?😀

**Comment by rkowase** [comment]
I wanna sent 5 quadrillion PRs ! 🔥

**Comment by takahirom** [comment]
💯

**Comment by takahirom** [comment]
Thanks 👍

**Pull request merged**